cla_backend/settings/circle.py

```diff
@@ -0,0 +1,26 @@
+import os
+from .testing import *
+
+DEBUG = False
+TEMPLATE_DEBUG = DEBUG
+
+ADMINS = (
+    ('CLA', 'cla-alerts@digital.justice.gov.uk'),
+)
+
+MANAGERS = ADMINS
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'cla_backend.apps.reports.db.backend',
+        'NAME': os.environ.get('DB_NAME', 'circle_test'),
+        'USER': os.environ.get('DB_USER', 'root'),
+        'PASSWORD': os.environ.get('DB_PASSWORD', ''),
+        'HOST': os.environ.get('DB_HOST', 'localhost'),
```

Comment:
I'm curious. We specify the `DATABASE_URL` in the CircleCI configuration (which can have user/password/database sections) but we define the arguments here again. Do we need both?

Reply:
I believe Django uses the keys `NAME`, `USER`, `PASSWORD` and `HOST`.

You're right about then environment variables, we don't necessarily have to specify defaults in this file, but I thought we may as well.

The DATABASE_URL is for convenience only in the Setup Postgres step, but I can see how it would break things if the env variables were changed. I'll scrap the DATABASE_URL and reference the env vars directly in that step.

Reply:
@rowlando you can use `DATABASE_URL` via this package though: https://github.com/kennethreitz/dj-database-url

👋

Reply:
Thanks @kerin 

I'll leave it to a Python developer to sort out. Apparently we're getting one soon!